CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ Fix a bug which would drop samples in remote-write if the sending flow stalled
 for longer than it takes to write one "WAL segment". How long this takes depends on the size
 of your Prometheus; as a rough guide with 10 million series it is about 2-3 minutes.
 
-* [BUGFIX] Remote-Write: stop dropping samples in catch-up #14446
+* [BUGFIX] Remote-write: stop dropping samples in catch-up #14446
```

Comment:
we can still remove this

Reply:
hm, I thought I did, sorry

Reply:
so, I don't think I touch this anymore in this diff, can you double check?